modules/mixins/browser-state.js

```diff
@@ -1,4 +1,17 @@
+var globalMouseUp = require("./util/global-mouseup");
```

Comment:
How about naming the module MouseUpListener and providing a simple method, subscribe, that takes a callback and returns an object with a `remove()` method. Then, BrowserStateMixin is not tightly coupled, and you don't have the magic of handleGlobalMouseUp being called across files.